packages/client/src/internals/fetchHelpers.ts

```diff
@@ -2,11 +2,10 @@ export function getWindow() {
   if (typeof window !== 'undefined') {
```

Comment:
Can we remove this one too?

Reply:
I tried getting rid of getWindow() all together, but there seemed to be an issue with `jest-environment-jsdom` not setting fetch on `globalThis` properly.
https://github.com/trpc/trpc/runs/8041790807?check_suite_focus=true#step:5:710

Reply:
Oh wait, no you're right. It's the tests that are wrong
https://github.com/trpc/trpc/blob/next/packages/server/test/interop/clientInternals.test.ts#L42-L45

Reply:
I'd love some help fixing those tests. Then everywhere we currently call `getWindow()` can just be replaced by `globalThis`.

Reply:
Shouldn't it still work if we add the property to `global`?